### PR TITLE
fix deprecations

### PR DESCRIPTION
**Pull request by abhijithch**
Fixes deprecations for `Julia v0.5`.

### CODE REVIEW

**Comment by tkelman** [comment]
Add the minimum `Compat` version requirement that introduced the features you're using.

**Comment by andreasnoack** [comment]
I'm not sure why this fails on 0.4. Maybe try
```
using Compat
import Compat.String
```
in the test file.

File: test/cipher_tests.jl [hunk 1]
```diff
@@ -84,17 +84,17 @@ willbebroken = "this is 16 (∀)" # case of length(::String) != 16
 @test length(willbebroken) == 14
 @test sizeof(willbebroken) == 16
 @test willbebroken.data == decrypt(dec, encrypt(enc, willbebroken))
-@test willbebroken == bytestring(decrypt(dec, encrypt(enc, willbebroken)))
+@test willbebroken == @compat String(decrypt(dec, encrypt(enc, willbebroken)))
```
Comment (tkelman) [inline test/cipher_tests.jl]: needs import, does not need `@compat`

Comment (tkelman) [inline test/cipher_tests.jl]: my mistake, these probably do actually need `@compat` for 0.3 support

**Comment by abhijithch** [comment]
I think I have made the necessary changes, could this be merged?

**Pull request merged**